pkg/registries/registries.go

```diff
@@ -124,9 +143,17 @@ func EditRegistriesConfig(config *sysregistriesv2.V2RegistriesConf, insecureRegi
 		return err
 	}
 	for _, mirrorSet := range mirrorSets {
+		// If source is a wildcard entry skip it as we don't currently support wildcards for mirror configuration
+		if strings.Contains(mirrorSet.Source, "*") {
```

Comment:
Why is it OK to _silently_ ignore user input? (BTW this could work well enough.)

Reply:
Fixed.
Moved validation to the MCO code, so if a user tries to use a wildcard entry for mirrors it will throw an error before we call EditRegistriesConfig.